src/Veldrid.Tests/ComputeTests.cs

```diff
@@ -2,23 +2,196 @@
+                "main"));
+
+            using ResourceLayout computeLayout = RF.CreateResourceLayout(new ResourceLayoutDescription(
+                new ResourceLayoutElementDescription("TextureToFill", ResourceKind.TextureReadWrite, ShaderStages.Compute),
+                new ResourceLayoutElementDescription("FillValueBuffer", ResourceKind.UniformBuffer, ShaderStages.Compute)));
+
+            using Pipeline computePipeline = RF.CreateComputePipeline(new ComputePipelineDescription(
+                computeShader,
+                computeLayout,
+                16, 16, 1));
+
+            using DeviceBuffer fillValueBuffer = RF.CreateBuffer(new BufferDescription((uint)Marshal.SizeOf<FillValueStruct>(), BufferUsage.UniformBuffer));
+
+            // Create our output texture.
+            using Texture computeTargetTexture = RF.CreateTexture(TextureDescription.Texture3D(
+                OutputTextureSize,
+                OutputTextureSize,
+                OutputTextureSize,
+                1,
+                PixelFormat.R32_G32_B32_A32_Float,
+                TextureUsage.Sampled | TextureUsage.Storage));
+
+            using TextureView computeTargetTextureView = RF.CreateTextureView(computeTargetTexture);
+
+            using ResourceSet computeResourceSet = RF.CreateResourceSet(new ResourceSetDescription(
+                computeLayout,
+                computeTargetTextureView,
+                fillValueBuffer));
+
+            using CommandList cl = RF.CreateCommandList();
+            cl.Begin();
+
+            cl.UpdateBuffer(fillValueBuffer, 0, new FillValueStruct(FillValue));
+
+            // Use the compute shader to fill the texture.
+            cl.SetPipeline(computePipeline);
+            cl.SetComputeResourceSet(0, computeResourceSet);
+            const uint GroupDivisorXY = 16;
+            cl.Dispatch(OutputTextureSize / GroupDivisorXY, OutputTextureSize / GroupDivisorXY, OutputTextureSize);
+
+            cl.End();
+            GD.SubmitCommands(cl);
+            GD.WaitForIdle();
+
+            // Read back from our texture and make sure it has been properly filled.
+            for (uint depth = 0; depth < computeTargetTexture.Depth; depth++)
+            {
+                RgbaFloat expectedFillValue = new RgbaFloat(new System.Numerics.Vector4(FillValue * (depth + 1)));
+                int notFilledCount = CountTexelsNotFilledAtDepth(GD, computeTargetTexture, expectedFillValue, depth);
+
+                if (notFilledCount == 0)
+                {
+                    // Expected behavior:
+                    Console.WriteLine($"All texels were properly set at depth {depth}");
+                }
+                else
+                {
+                    // Unexpected behavior:
+                    uint totalTexels = computeTargetTexture.Width * computeTargetTexture.Height;
+                    throw new Exception($"{notFilledCount} of {totalTexels} texels were not properly set at depth {depth}");
```

Comment:
Use Xunit assertions so the tests fail in a standard way instead of throwing a generic Exception.